doc/htmldoc/nest_behavior/built-in_timers.rst

```diff
@@ -82,11 +82,11 @@ available as kernel attributes:
 +================================+==================================+==================================+
 |``time_gather_target_data``     |Cumulative time for communicating |``time_communicate_prepare``      |
 |                                |connection information from       |                                  |
-|				 |postsynaptic to presynaptic side  |                                  |
+|				                 |postsynaptic to presynaptic side  |                                  |
 +--------------------------------+----------------------------------+----------------------------------+
 |``time_communicate_target_data``|Cumulative time for core MPI      |``time_gather_target_data``       |
 |                                |communication when gathering      |                                  |
-|				 |target data                       |                                  |
+|			                	 |target data                       |                                  |
 +--------------------------------+----------------------------------+----------------------------------+
 |``time_update``                 |Time for neuron update            |``time_simulate``                 |
 +--------------------------------+----------------------------------+----------------------------------+
```

Comment:
Documentation on "time_mpi_synchronization" needed. E.g. "Time for synchronization of MPI processes during spike communication"